api/krusty/fnplugin_test.go

```diff
@@ -629,3 +631,148 @@ metadata:
   name: env
 `)
 }
+
+func TestFnContainerMounts(t *testing.T) {
+	skipIfNoDocker(t)
+
+	th := kusttest_test.MakeHarness(t)
+	o := th.MakeOptionsPluginsEnabled()
+	fSys := filesys.MakeFsOnDisk()
+	b := krusty.MakeKustomizer(&o)
+	tmpDir, err := filesys.NewTmpConfirmedDir()
+	assert.NoError(t, err)
+
+	path, err := os.Getwd()
+	assert.NoError(t, err)
+	chartPath := filepath.Join(path, "testdata", "charts")
+
+	assert.NoError(t, fSys.WriteFile(filepath.Join(tmpDir.String(), "kustomization.yaml"), []byte(`
+generators:
+  - |-
+    apiVersion: v1alpha1
+    kind: RenderHelmChart
+    metadata:
+      name: demo
+      annotations:
+        config.kubernetes.io/function: |
+          container:
+            image: gcr.io/kpt-fn/render-helm-chart:v0.1.0
+            mounts:
+            - type: "bind"
+              src: "`+chartPath+`"
+              dst: "/tmp/charts"
+    helmCharts:
+    - name: helloworld-chart
+      releaseName: test
+      valuesFile: /tmp/charts/helloworld-values/values.yaml
+`)))
+	m, err := b.Run(
+		fSys,
+		tmpDir.String())
+	if utils.IsErrTimeout(err) {
+		// Don't fail on timeouts.
+		t.SkipNow()
+	}
```

Comment:
Suggest not doing this on test since it can either introduce flakiness or bypass covering for corner cases. Do you know which step may cause the timeout? would it be possible to mock it?

Reply:
Removed! I don't think it's necessary. Mistakenly copied the lines from another test, thanks for the catch.